ampl-converter/src/main/java/com/powsybl/ampl/converter/AmplNetworkWriter.java

```diff
@@ -1311,9 +1420,9 @@ private void writeShunts(AmplExportContext context) throws IOException {
                      new Column("sections count"))) {
             List<String> skipped = new ArrayList<>();
             for (ShuntCompensator sc : network.getShuntCompensators()) {
-                if (sc.getModelType() == ShuntCompensatorModelType.NON_LINEAR) {
+/*                if (sc.getModelType() == ShuntCompensatorModelType.NON_LINEAR) {
                     throw new PowsyblException("Non linear shunt compensator not yet supported");
-                }
+                }*/
```

Comment:
You can delete these lines (not comment them)

Reply:
Ok!